packages/api/src/__tests__/e2e/mapi/parts/consolidated.test.part.ts

```diff
@@ -348,6 +348,38 @@ export function runConsolidatedTests(e2e: E2eContext) {
     resetWebhook(e2e);
     expect(true).toBeTrue();
   });
+
+  /*************************************************************
+   * Consolidated Query - Don't get WH when fromDashboard is true
+   *************************************************************/
```

Comment:
no need for the comment, the test name is pretty clear

Reply:
Going to leave it - its a good separation from the rest of the tests anyways

Reply:
Those things get outdated over time. We discussed and decided that comments make sense when they explain why we're doing something that way, not what the code does - and certainly not to separate sections of the code - [reference](https://www.notion.so/metriport/Development-Best-Practices-bb3574d3c83d46dc9914a9e5c5542ca7?pvs=4#fb560de23853473f88ea7011630eb6b0).